### DISCUSSION TITLE
Does Creator Kit require a premium license?

### DISCUSSION THREAD

**Discussion by RyanStone1** [microsoft/powercat-creator-kit#282] [Q&A] [answered]
It appears so but I have yet to find any documentation to confirm on this.

**Comment by yaseralhosani** [comment, marked as answer]
Hi @RyanStone1,

Power Apps canvas did not require a premium license for basic functionality as long as you can import code component framework. Working great in production and developer environments.

Power Apps for Microsoft Dataverse Team has limited capabilities, I tried to import and I have not found the answer yet. More likely, it's impossible to import code components. The rest should be fine.

**Reply by RyanStone1** [reply]
Thank you!  We discovered later the issue was coming from reading Environment Variables rather than Creator Kit.  But good to get confirmation on the latter also.
